tests/unit_test/channels/whatsapp_broadcast_test.py

```
@@ -565,4 +566,131 @@ def test_log_failed_messages():
+    event_id = "test_event_id"
+    reference_id = "test_reference_id"
+
+    whatsapp_broadcast = WhatsappBroadcast(bot, user, config, event_id, reference_id)
+
+    async def mock_send_template_message_async(*args, **kwargs):
+        return True, 200, {}
+
+    async def mock_execute_rule(*args, **kwargs):
+        return [{"text": '{"param": "value"}'}], None
+
+    with patch.object(WhatsappBroadcast, '_WhatsappBroadcast__get_client', return_value=MagicMock()) as mock_get_client, \
```

Ruff (0.8.2)

Local variable `mock_get_client` is assigned to but never used

Remove assignment to unused variable `mock_get_client`

(F841)